fn/common/api.go

```diff
@@ -0,0 +1,28 @@
+package common
+
+import (
+	"os"
+
+	"crypto/tls"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	fnclient "github.com/iron-io/functions_go/client"
+	"net/http"
```

Comment:
Nit: Please consider to separate imports of standard packages in one group.

Reply:
Ok, thanks ! I'll group them in one group.